org.eclipse.lsp4j/src/main/java/org/eclipse/lsp4j/InsertTextMode.java

```diff
@@ -0,0 +1,62 @@
+/******************************************************************************
+ * Copyright (c) 2020 TypeFox and others.
+ * 
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0,
+ * or the Eclipse Distribution License v. 1.0 which is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ * 
+ * SPDX-License-Identifier: EPL-2.0 OR BSD-3-Clause
+ ******************************************************************************/
+package org.eclipse.lsp4j;
+
+import com.google.common.annotations.Beta;
+
+/**
+ * How whitespace and indentation is handled during completion
+ * item insertion.
+ *
+ * Since 3.16.0
+ */
+@Beta
```

Comment:
Why beta?

Reply:
This PR is 2 weeks old which is before it was finalized. You will see a lot of that annotation. I will have to remove all of those now.